src/smc-webapp/buttonbar.coffee

```diff
@@ -351,14 +355,18 @@ exports.commands =
             wrap :
                 left  : "\n#### "
                 right : ""
-        format_code :  # FUTURE: I think indentation is probably nicer?  on single line ` is nicer.
+        format_code :
             wrap :
-                left  : '\n```'
-                right : '\n```\n'
+                left  : '    '
+                right : ''
+                multi : true
+                space : false
         indent :
             wrap :
-                left  : "\n> "
-                right : ""
+                left  : '>'
+                right : ''
+                multi : true
+                space : true
```

Comment:
I'm not sure what's going on, but quoting seems slightly off.
![highlight](https://cloud.githubusercontent.com/assets/618575/20848802/8cd0089e-b888-11e6-8d56-04be72a442ec.png)
![quoted](https://cloud.githubusercontent.com/assets/618575/20848803/8dc9ee40-b888-11e6-9d1a-3ba5a522fc7f.png)